bin/add-users-and-roles.sh

```diff
@@ -10,11 +10,11 @@ export CCD_CONFIGURER_S2S_SECRET=AAAAAAAAAAAAAAAC
 
 dir=$(dirname ${0})
 
-${dir}/../kubernetes/configurer/utils/idam-create-caseworker.sh damian@swansea.gov.uk caseworker,caseworker-publiclaw,caseworker-publiclaw-solicitor
-${dir}/../kubernetes/configurer/utils/idam-create-caseworker.sh kurt@swansea.gov.uk caseworker,caseworker-publiclaw,caseworker-publiclaw-solicitor
-${dir}/../kubernetes/configurer/utils/idam-create-caseworker.sh james@swansea.gov.uk caseworker,caseworker-publiclaw,caseworker-publiclaw-solicitor
-${dir}/../kubernetes/configurer/utils/idam-create-caseworker.sh sam@hillingdon.gov.uk caseworker,caseworker-publiclaw,caseworker-publiclaw-solicitor
-${dir}/../kubernetes/configurer/utils/idam-create-caseworker.sh siva@hillingdon.gov.uk caseworker,caseworker-publiclaw,caseworker-publiclaw-solicitor
-${dir}/../kubernetes/configurer/utils/idam-create-caseworker.sh hmcts-admin@example.com caseworker,caseworker-publiclaw,caseworker-publiclaw-courtadmin
+${dir}/../kubernetes/configurer/utils/idam-create-caseworker.sh damian@swansea.gov.uk caseworker,caseworker-publiclaw,caseworker-publiclaw-solicitor "(local-authority)"
+${dir}/../kubernetes/configurer/utils/idam-create-caseworker.sh kurt@swansea.gov.uk caseworker,caseworker-publiclaw,caseworker-publiclaw-solicitor "(local-authority)"
+${dir}/../kubernetes/configurer/utils/idam-create-caseworker.sh james@swansea.gov.uk caseworker,caseworker-publiclaw,caseworker-publiclaw-solicitor "(local-authority)"
+${dir}/../kubernetes/configurer/utils/idam-create-caseworker.sh sam@hillingdon.gov.uk caseworker,caseworker-publiclaw,caseworker-publiclaw-solicitor "(local-authority)"
+${dir}/../kubernetes/configurer/utils/idam-create-caseworker.sh siva@hillingdon.gov.uk caseworker,caseworker-publiclaw,caseworker-publiclaw-solicitor "(local-authority)"
+${dir}/../kubernetes/configurer/utils/idam-create-caseworker.sh hmcts-admin@example.com caseworker,caseworker-publiclaw,caseworker-publiclaw-courtadmin "(hmcts-admin)"
```

Comment:
CAFCASS is missing.

Also do same in `kubernetes/configurer/configure-ccd.sh`.